docs/pages/application-access/okta/scim-only.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: Installing SCIM-only Okta integration
+description: How to install a SCIM-only Okta integration
+---
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+- [Authentication With Okta as an SSO Provider](../../access-controls/sso/okta)
+
+
+## How it works
+User provisioning (and de-provisioning) with SCIM requires two Teleport
+components working together:
+
+ * A SAML Connector that provides SSO login to Teleport for upstream Okta users
+ * A SCIM integration that provisions and de-provisions Teleport user accounts
+   in response to changes in the upstream Okta organization
+
+Both of these Teleport components rely on an Okta SAML Application to act as the
```

Comment:
```suggestion
Both of these Teleport components rely on an Okta SAML application to act as the
```
As far as I know, we don't tend to capitalize the names of protected resources, so we would lowercase "application".